agent/src/agent/data_extractor/cacti/cacti.py

```diff
@@ -40,8 +40,14 @@ def extract_metrics(pipeline_: Pipeline, start: str, end: str, step: str) -> lis
 
             base_metric = {
                 'target_type': 'gauge',
-                'properties': _extract_dimensions(item, graph, cache.hosts,
-                                                  pipeline_.config['add_graph_name_dimension']),
+                'properties': _extract_dimensions(
+                    item,
+                    graph,
+                    cache.hosts,
+                    str(local_graph_id),
+                    pipeline_.config['add_graph_name_dimension'],
+                    pipeline_.config['add_graph_id_dimension'],
```

Comment:
i think both of them should be `pipeline_.config.get('add_graph_id_dimension', False)`